src/main/game/entities/units/UnitNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package game.entities.units;
+
+/**
+ * Created by khariollivierre on 2/25/17.
+ */
+public class UnitNotFoundException extends Throwable {
```

Comment:
this should extend Exception not Throwable. Throwable is the super class for both Exception and Error